projects/plugins/crm/js/jpcrm-admin-funnel.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Draws a simple funnel
+ * 
+ * Inspired in part by https://codepen.io/tylerlwsmith/pen/RwNZKgp
+ * 
+ * @param funnel_data array of funnel data (count, backfill_count, contact_status, and link)
+ * @element HTML element to turn into a funnel
+ **/
+
+function jpcrm_build_funnel(data, funnel_element) {
+	// something's wrong, so stop
+	if (!element || !data) return;
+
+	const COLORS = [ '#000', '#222', '#333', '#035d88', '#0073aa', '#00a0d2' ];
+
+	// show legend if desired
+	const SHOW_FUNNEL_LEGEND = false;
+
+	num_of_funnel_sections = data.length;
+	funnel_height = num_of_funnel_sections * 50;
```

Comment:
I think that not using `let` here declares these two variables in the global scope. I'm not sure if this is a problem or not, I just want to highlight this.